### ISSUE TITLE
Turn off autosaving if file is (or becomes) too large

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#6321]
I just produced a nice graph with the lattice package.  Here it is for the data from Germany:

![image](https://user-images.githubusercontent.com/11226469/111949775-395f1100-8ad9-11eb-80a0-3ae9dad4986e.png)

It is very nice.  But it seems to be so large that the backing up every 10 minutes becomes impossible. 

I have turned it off.  But it raises a more general point that we are now able to handle large files in R-Instat.  I suggest we consider adding an option to the backing up system that turns it off if the file is too large - and we can define "too large" in the options.  In that case it may become too large during use of R-Instat when we add data frames.  When it goes over a limit, then the backing up system sends a message to the output window - perhaps even every time it would usually backup, to say it isn't because the data book is now xxx mbytes.

This is not urgent

**Comment by dannyparsons** [comment]
To know the size, we'd still have to auto-save it once and then check the size before the next auto-save. So if it's crashing because of the size, I don't think this will really solve your issue.

Unfortunately, we can't timeout an R command, otherwise we could stop after a certain time taken.

I think solving the core issue of the size of these graphs when saved will be the best help.

**Comment by rdstern** [comment]
@dannyparsons what will your solution mean for existing files - that have models and graphs?  I assume we can still open them?

**Comment by dannyparsons** [comment]
Yes, I believe existing files created before this change will still function in the same way. But please test this, I just opened the PR https://github.com/africanmathsinitiative/R-Instat/pull/6344

**Comment by lilyclements** [comment]
Autosaving can be manually turned off. In addition, we have discussed elsewhere about handling cases like graphics becoming too large - e.g., #9448, #3842
